tools/wpt/run.py

```diff
@@ -223,6 +223,7 @@ def setup_kwargs(self, kwargs):
             else:
                 raise WptrunError("Unable to locate or install chromedriver binary")
 
+
```

Comment:
Accidental change? Just a whitespace change in a file looks weird.

Reply:
It's actually intentional, but only for styles... We should have two blank lines between top-level classes/functions according to PEP8. And this is the only offence in this file which I happened to spot.